src/Features/CSharp/Portable/QuickInfo/CSharpSemanticQuickInfoProvider.cs

```diff
@@ -151,6 +152,10 @@ protected override NullableFlowState GetNullabilityAnalysis(SemanticModel semant
         {
             return null;
         }
+        if (document.IsRazorDocument())
```

Comment:
don't use the extension method from EA.Razor.Api namespace, use the one under `Microsoft.CodeAnalysis.Host` directly instead
https://github.com/dotnet/roslyn/blob/main/src/Workspaces/Core/Portable/Workspace/Host/DocumentService/Extensions.cs